core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -96,7 +97,12 @@ class TestSuiteWriter {
 
         //catch any sorting problems (see NPE is SortingHelper on Trello)
         val tests = try {
-            testSuiteOrganizer.sortTests(solution, config.customNaming)
+            // NEED to check with Andrea, skip to sort RPC for the moment
```

Comment:
add `TODO` on this comment

Reply:
yes, in future we need to handle test suite split for all types, including RPC ang GQL